proto/commonfate/control/config/v1alpha1/slack_alert.proto

```diff
@@ -42,6 +43,8 @@ message SlackAlert {
   bool disable_interactivity_handlers = 8;
   // if specified, it will notify the user the set time before the access expires
   google.protobuf.Duration notify_expiry_in_seconds = 9;
+  // Configuration for user notifications, allowing opt-in and opt-out for specific DMs
+  UserNotificationSettings user_notification_settings = 10;
```

Comment:
@ckluy31 we should make these preferences per-user, rather than global. I am thinking that in the web app a user can go to their profile and configure their notification settings.